### DISCUSSION TITLE
Question about Ad unit id

### DISCUSSION THREAD

**Discussion by wangdong20** [Poing-Studios/godot-admob-editor#24] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I found I can edit ad unit id in Admob editor, there are default test unit id with standard key. What does the standard key mean? Are there options for key rather than standard? The standard key can only be edited rather than select one option. How do I test it with real unit id export as no debug?

**Describe the solution you'd like**
I just edited the default test unit id to real unit id, but I will lose the default test unit id when I want to export project as debug.

**Describe alternatives you've considered**
Is there a way to keep the test unit id and add a new real unit id in the editor and the editor will select to correct unit id based on the selection on debug or not when export it?

**Additional context**
What I talked about is the unit id for each ad in Admob editor in Godot.

**Comment by gumaciel** [comment, marked as answer]
Hey sorry for delay.

> there are default test unit id with standard key. What does the standard key mean?

A: "standard" is to be a "global" ad, if you don't want to create multiple ad ids, you can just use "standard", if you don't pass any parameter in the load_SOMEADTYPE() methods, it will try to load the " standard"

https://github.com/Poing-Studios/godot-admob-editor/blob/master/addons/admob/src/singletons/MobileAds.gd#L8-L22

> Are there options for key rather than standard? 

A: yes, just create other ads on admob.com and create in editor, then put it to load_SOMEADTYPE("your_key")
![image](https://user-images.githubusercontent.com/20030153/157749737-29b4eefa-1348-4f6b-ac82-cc4bd05f2eb2.png)

Why do this: 

```
There are a few reasons:
1. You want to set different eCPM price points for the same ad unit. In Unity today, you must set up a new placement for the same ad unit if you want different eCPM price points.
2. If you have multiple points to surface an ad that are the same type but in different points in the game.
```

http://answers.unity.com/answers/1681925/view.html

> The standard key can only be edited rather than select one option. 

no, you can edit if you have it or several others

>How do I test it with real unit id export as no debug?

Currently when you export the application in debug mode, it is already put to test the ads, regardless of whether they are for testing or release, we do this: https://developers.google.com/admob/android/test-ads#add_your_test_device_programmatically

We can put an option in the "Debug" tab to "Test with Demo Ids only", what you think?

**Reply by wangdong20** [reply]
I think it will be better that don't provide such option "Test with Demo Ids only" in "Debug" tab. When in debug mode, app will be test with demo id for ad as default. If you put an option, then developer can close it, but it doesn't make sense that app will run with real unit id in debug mode.

**Comment by gumaciel** [comment]
I'm going to convert this Issue to Discussion because it looks like a question thread

**Comment by gumaciel** [comment]
Oh,

>Is there a way to keep the test unit id and add a new real unit id in the editor and the editor will select to correct unit id based on the selection on debug or not when export it?

Yes but i think this will be the solution that you are thinking:

`We can put an option in the "Debug" tab to "Test with Demo Ids only", what you think?`
